manifests/params.pp

```diff
@@ -164,4 +166,13 @@
   $certificate_whitelist      = [ ]
   # change to this to only allow access by the puppet master by default:
   #$certificate_whitelist      = [ $::servername ]
+
+  # Get the parameter name for the database connection pool tuning
+  if $puppetdb_version in ['latest','present'] or versioncmp($puppetdb_version, '4.0.0') >= 0 {
+    $database_max_pool_size_setting_name = 'maximum-pool-size'
+  } elsif versioncmp($puppetdb_version, '3.2.0') >= 0 {
```

Comment:
No problem I've got you covered there, I've verified this on 2.8.7 so it should be ok for the whole of the 2.8 series (which is as far back as we support anyways).